### PR TITLE
Use os separator instead of hard coded /

### PR DESCRIPTION
**Pull request by markcurtis1970**
Fixes the issue with windows based systems which don't understand '/'

### CODE REVIEW

File: logdna/configs.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from os import path
+from os import sep
 
-with open("%s/VERSION" % path.abspath(path.dirname(__file__))) as f:
+with open("{p}{s}VERSION".format(p=path.abspath(path.dirname(__file__)),s=sep)) as f:
```
Comment (esatterwhite) [inline logdna/configs.py:4]: needs a white space after the `,` 
`, s=sep`

File: setup.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+from os import sep
 
 # read the contents of your README file
 this_directory = path.abspath(path.dirname(__file__))
 
 with open(path.join(this_directory, 'README.md'), 'rb') as f:
     long_description = f.read().decode('utf-8')
 
-with open("%s/logdna/VERSION" % this_directory) as f:
+with open("{p}{s}logdna{s}VERSION".format(p=this_directory,s=sep)) as f:
```
Comment (esatterwhite) [inline setup.py:11]: ditto on the comman `, s=sep`

File: logdna/configs.py (hunk 1 above)
Comment (esatterwhite) [reply logdna/configs.py:4]: This line might be a few too many chars long as well.

Comment (markcurtis1970) [reply logdna/configs.py]: thanks, corrected now

File: setup.py (hunk 2 above)
Comment (markcurtis1970) [reply setup.py]: thanks, corrected now

File: logdna/configs.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 from os import path
+from os import sep
```
Comment (esatterwhite) [inline logdna/configs.py:2]: small, but these can be combined

```python
from os import path, sep
```

File: setup.py [hunk 4]
```diff
@@ -1,13 +1,16 @@
 from setuptools import setup
 from os import path
+from os import sep
```
Comment (esatterwhite) [inline setup.py:3]: Should combine these two imports

```python
from os import path, sep
```

**Review by esatterwhite — COMMENTED** [review]
@markcurtis1970 just 2 import fixes, and this looks good togo

**Review by esatterwhite — APPROVED** [review]

**Review by mdeltito — APPROVED** [review]

**Review by esatterwhite — APPROVED** [review]

**Pull request merged**